sqlglot/parser.py

```diff
@@ -6485,12 +6485,7 @@ def _parse_alter_table_set(self) -> exp.AlterSet:
 
         return alter_set
 
-    def _parse_alter(self) -> exp.AlterTable | exp.Command:
-        start = self._prev
-
-        if not self._match(TokenType.TABLE):
-            return self._parse_as_command(start)
-
+    def _parse_alter_table(self, start) -> exp.AlterTable | exp.Command:
```

Comment:
Maybe we can instead make a set of tokens, `ALTERABLES`, starting with `TABLE` and `VIEW` and then do:

```python
if not self._match_set(self.ALTERABLES):
    return self._parse_as_command(start)
...
```

This way we'll avoid adding the `start` argument everywhere.

Reply:
Sounds great.